// @applitools/eyes/dist/enums/IosDeviceName.js
"use strict";
/* GENERATED FILE */
Object.defineProperty(exports, "__esModule", { value: true });
exports.IosDeviceNameEnum = void 0;
var IosDeviceNameEnum;
(function (IosDeviceNameEnum) {
    IosDeviceNameEnum["iPad_7"] = "iPad (7th generation)";
    IosDeviceNameEnum["iPad_9"] = "iPad (9th generation)";
    IosDeviceNameEnum["iPad_Air_4"] = "iPad Air (4th generation)";
    /** @deprecated use {@link iPad_Pro_12_9_inch_3 } instead */
    IosDeviceNameEnum["iPad_Pro_3"] = "iPad Pro (12.9-inch) (3rd generation)";
    IosDeviceNameEnum["iPad_Pro_12_9_inch_3"] = "iPad Pro (12.9-inch) (3rd generation)";
    /** @deprecated use {@link iPad_Pro_11_inch_4 } instead */
    IosDeviceNameEnum["iPad_Pro_4"] = "iPad Pro (11-inch) (4th generation)";
    IosDeviceNameEnum["iPad_Pro_11_inch_4"] = "iPad Pro (11-inch) (4th generation)";
    IosDeviceNameEnum["iPhone_8"] = "iPhone 8";
    IosDeviceNameEnum["iPhone_8_Plus"] = "iPhone 8 Plus";
    IosDeviceNameEnum["iPhone_X"] = "iPhone X";
    IosDeviceNameEnum["iPhone_XR"] = "iPhone XR";
    /** @deprecated use {@link iPhone_Xs } instead */
    IosDeviceNameEnum["iPhone_XS"] = "iPhone Xs";
    IosDeviceNameEnum["iPhone_Xs"] = "iPhone Xs";
    IosDeviceNameEnum["iPhone_11"] = "iPhone 11";
    IosDeviceNameEnum["iPhone_11_Pro_Max"] = "iPhone 11 Pro Max";
    IosDeviceNameEnum["iPhone_11_Pro"] = "iPhone 11 Pro";
    IosDeviceNameEnum["iPhone_12"] = "iPhone 12";
    IosDeviceNameEnum["iPhone_12_mini"] = "iPhone 12 mini";
    IosDeviceNameEnum["iPhone_12_Pro_Max"] = "iPhone 12 Pro Max";
    IosDeviceNameEnum["iPhone_12_Pro"] = "iPhone 12 Pro";
    IosDeviceNameEnum["iPhone_13"] = "iPhone 13";
    IosDeviceNameEnum["iPhone_13_Pro_Max"] = "iPhone 13 Pro Max";
    IosDeviceNameEnum["iPhone_13_Pro"] = "iPhone 13 Pro";
    IosDeviceNameEnum["iPhone_14"] = "iPhone 14";
    IosDeviceNameEnum["iPhone_14_Pro_Max"] = "iPhone 14 Pro Max";
    IosDeviceNameEnum["iPhone_15"] = "iPhone 15";
    IosDeviceNameEnum["iPhone_15_Pro_Max"] = "iPhone 15 Pro Max";
})(IosDeviceNameEnum || (exports.IosDeviceNameEnum = IosDeviceNameEnum = {}));
